extensions/notebook/src/extension.ts

```diff
@@ -23,7 +23,7 @@ type ChooseCellType = { label: string, id: CellType };
 export async function activate(extensionContext: vscode.ExtensionContext): Promise<IExtensionApi> {
 	const appContext = new AppContext(extensionContext);
 	const createBookPath: string = path.posix.join(extensionContext.extensionPath, 'resources', 'notebooks', 'JupyterBooksCreate.ipynb');
-	extensionContext.subscriptions.push(vscode.commands.registerCommand('bookTreeView.openBook', (bookPath: string, openAsUntitled: boolean, urlToOpen?: string) => openAsUntitled ? providedBookTreeViewProvider.openBook(bookPath, urlToOpen, true) : bookTreeViewProvider.openBook(bookPath, urlToOpen, true)));
+	extensionContext.subscriptions.push(vscode.commands.registerCommand('bookTreeView.openBook', (bookPath: string, openAsUntitled: boolean, urlToOpen?: string, isNotebook?: boolean) => openAsUntitled ? providedBookTreeViewProvider.openBook(bookPath, urlToOpen, true, isNotebook) : bookTreeViewProvider.openBook(bookPath, urlToOpen, true, isNotebook)));
```

Comment:
Why wouldn't we call bookTreeView.openNotebook instead if it's a notebook? It isn't a terrible idea to have a single command be able to handle both - but if we do then we should just get rid of the other and have everything go through the one command IMO.

Reply:
yeah, I agree. OpenBook is not supposed to be used for opening notebook(s). But we're reusing openBook for opening items in a folder, hence the parameter. Added it for clarity.

Reply:
So, is the issue that the OpenNotebookFolder command only opens Books?